### PR TITLE
[WIP] add fixit draft/stubs

### PR DESCRIPTION
**Pull request by bsweger**
**THIS IS IN PROGRESS**

Started adding some of the fixit content we've been discussing in various docs and in Trello.

Will continue fleshing these out but right now am especially looking for feedback re: where on the hub these should live. Right now it's hub/fixit/.

### CODE REVIEW

File: pages/fixit/about.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+* The goal of a fixit is broader: to get something to the next plateau rather than to implement specific feature sets.
+
+## How Does a Fixit Work?
+
+1. Someone identifies a process that could benefit from a concentrated effort (for example, on-boarding) and decides to organizer a fixit.
+2. The fixit organizers compile a list of to-dos and assemble them in one place. These could be:
+    * GitHub issues
+    * Trello tasks
+    * Surveys
+2. At the appointed time, fixit participants collectively attack the to-do list, with individuals submitting a record of what they did. It's the organizers' job to make that last part easy.
```
Comment (mbland) [inline pages/fixit/about.md]: This is the second `2.`, but I guess they could be any number, as Markdown just does the right thing.

File: pages/fixit/about.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* provide a safe space for people to learn 18F tools
+
+## Who Should Fixit?
+
+Everyone at 18F can (and should!) contribute to a fixit. No special ops, code, or design skills necessary.
+
+For more details:
+
+* [Fixit planning guide](guides/planning/)
+* [Fixit participation guide](guides/participation)
```
Comment (mbland) [inline pages/fixit/about.md]: Add a trailing slash just for consistency's sake.

File: pages/fixit/guides/planning.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* Set a date and invite people. Include a link to the [Fixit Overview](/fixit)
+
+### Two Weeks Before Fixit
+* Identify candidate fixit tasks
+* Create a single place that people can use to find tasks during the Fixit
+* Update the invite with a link to the task list
+
+### One Week Before Fixit
+* Make sure the [Fixit Participant Guide](/fixit/guides/participation) is current (or created)
+
```
Comment (mbland) [inline pages/fixit/guides/planning.md]: Maybe add a few placeholders:

- One Day Before Fixit
- Fixit Day!
- One Week After Fixit

File: pages/fixit/onboarding.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+permalink: /fixit/onboarding
+---
+# Onboarding Fixit
+
+The theme for 18F's first official [fixit](/fixit) is onboarding, and it will culminate on Friday, April 17th.
```
Comment (mbland) [inline pages/fixit/onboarding.md]: April 17, 2015.

File: pages/fixit/onboarding.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+### Fixit Day: April 17th
+
+We'll congregate to tackle the collected issues (and create new ones if necessary), reporting along the way. We'll likely use a Google form to track what gets done. This tracking is a critical step, since we want to collect fixit metrics.
+
+### Post-Fixit
+
+The fixit organizers will survey participants, sift through the collected metrics, and report back:
+
+* How many issues did we fix?
```
Comment (mbland) [inline pages/fixit/onboarding.md]: - How many people participated?
- What did the fixit help accomplish, for individuals, projects, working groups and guilds?
- What is now possible after the fixit that wasn't possible before?

**Comment by mbland** [comment]
This is lovely, @bsweger! Just a few minor comments.

File: pages/fixit/about.md (hunk 1 above)
Comment (bsweger) [inline pages/fixit/about.md]: Yeah, but I can't let that stand, even if Markdown does the right thing.

**Pull request merged**